ihatemoney/static/css/main.css

```diff
@@ -429,7 +429,6 @@ tr.payer_line .balance-name {
   position: absolute;
   top: 4.5rem;
   width: 100%;
-  pointer-events: none;
```

Comment:
why ?

Reply:
I removed this to prevent the situation you described above where clicking it does not trigger any action. if you try it in a incognito tab / safari it may work (I had the same issue while testing). in any case, I see what you mean about making the option more explicit. I will take a look at making these adjustments, thank you for the feedback